layers/meta-resin-beaglebone/recipes-bsp/u-boot/u-boot-ti-staging/beaglebone-integrate-with-resin-configuration.patch

```diff
@@ -0,0 +1,52 @@
+From 3e4294678099510ca978bd0e01f8f6132615def7 Mon Sep 17 00:00:00 2001
+From: Andrei Gherzan <andrei@resin.io>
+Date: Thu, 2 Mar 2017 00:45:56 +0100
+Subject: [PATCH] am335x_evm.h: Integrate with resin environment configuration
+
+Beaglebone machine specific integration of resin environment configuration.
+
+Signed-off-by: Andrei Gherzan <andrei@resin.io>
+
+Upstream-Status: Inappropriate [configuration]
+---
+ include/configs/am335x_evm.h      | 6 ++++--
+ include/configs/ti_armv7_common.h | 4 ++--
+ 2 files changed, 6 insertions(+), 4 deletions(-)
+
+diff --git a/include/configs/am335x_evm.h b/include/configs/am335x_evm.h
+index e1e89c4..b7af3ad 100644
+--- a/include/configs/am335x_evm.h
++++ b/include/configs/am335x_evm.h
+@@ -96,8 +96,11 @@
+ 	"fi;"	\
+ 	"run findfdt; " \
+ 	"run init_console; " \
+-	"run envboot; " \
+-	"run distro_bootcmd"
++	"setenv resin_kernel_load_addr ${loadaddr}; "\
++	"run resin_set_kernel_root; "\
++	"setenv mmcdev ${resin_mmcdev}; "\
```

Comment:
I think in this patch you are supposed to actually define resin_mmcdev. Above you redefine mmcdev based on a non-existing resin_mmcdev?